sale_order_line_date/README.rst

```diff
@@ -14,12 +14,12 @@ of the lines change to match.
 Usage
 =====
 
-Create a Quotation or a Sales Order and fill the requested date in the sale
+Create a Quotation or a Sales Order and it fills the requested date in the sale
```

Comment:
not it is not if it starts empty:

1. Create new quotation with a line with not required date
![image](https://cloud.githubusercontent.com/assets/23449160/26591117/edef5970-455c-11e7-9ae5-81e1c71ae683.png)

2. Change the required date for the SO:
![image](https://cloud.githubusercontent.com/assets/23449160/26591168/10181f00-455d-11e7-841e-91ea81ed8892.png)

3. Line wasn't updated:
![image](https://cloud.githubusercontent.com/assets/23449160/26591201/28cdcebe-455d-11e7-8007-d76576d51c2a.png)